js/apps/account-ui/src/api.ts

```diff
@@ -103,5 +104,11 @@ async function getAccessToken() {
     keycloak.login();
   }
 
+  const locale = keycloak.tokenParsed?.locale;
+  i18n.changeLanguage(locale, (error) => {
```

Comment:
Changing the language here after the user has signed in might cause an additional request if the configured locale is different than the default one.  We could instead provide the locale from the environment, this information is [already exposed](https://github.com/keycloak/keycloak/blob/2bb9992eb7be4577fe05bddcb5a9e3f66203c29d/services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java#L126) as data to the template, but still needs to be added to the [environment JSON](https://github.com/keycloak/keycloak/blob/2bb9992eb7be4577fe05bddcb5a9e3f66203c29d/js/apps/account-ui/pom.xml#L99-L119).